src/mqt/qecc/codes/core/stabilizer_code.py

```
@@ -293,23 +305,26 @@ def get_trivial_code(cls, n: int) -> StabilizerCode:
         x_logicals: list[str] = ["I" * i + "X" + "I" * (n - i - 1) for i in range(n)]
         return StabilizerCode([], distance=1, z_logicals=z_logicals, x_logicals=x_logicals, n=n)
 
-    def compute_logical_ops(self) -> None:
+    def compute_logical_ops(self) -> tuple[PauliTableau, PauliTableau]:
```

Ruff (0.15.21)

[warning] Too many branches (18 > 12)

(PLR0912)